src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -336,12 +345,20 @@ updateShellAndSC(unsigned int  boardIdx, DSAInfo& candidate, bool& reboot)
   if (!same_bmc) {
     std::cout << "Updating SC firmware on device[" << flasher.sGetDBDF() <<
       "]" << std::endl;
-    update_SC(boardIdx, candidate.file);
+    try {
+      update_SC(boardIdx, candidate.file);
+    } catch (const xrt_core::error& e) {
+      std::cout << "NOTE: Skipping SC flash. " << e.what() << std:: endl;
```

Comment:
It is always good to look at the error being caught if it is being ignored.  If it isn't what is expected then it should be re-thrown.